src/clients/storage/StorageClientBase-inl.h

```diff
@@ -240,16 +240,18 @@ void StorageClientBase<ClientType, ClientManagerType>::getResponseImpl(
     DCHECK(!!ioThreadPool_);
     evb = ioThreadPool_->getEventBase();
   }
+  auto reqPtr = std::make_shared<std::pair<HostAddr, Request>>(std::move(request.first),
```

Comment:
Would you explain more about why we need to use smart pointer here? `collectResponse` need to send request to multiple storage, but `getResponse` does not need to.

Reply:
RPC make a copy of requests, request can be moved after send and noone will use. While local function call need requests to stay where it is until function call is over.

Reply:
Maybe I should isolute this with MACRO.

Reply:
Ah I get your your point. Because in stand alone mode, the `client` is the `GraphStorageLocalServer`, so the request need to out-live until the function call is over.

I don't quite remember the `request` parameter type in `remoteFunc`. If it only accepts const reference, maybe we we do need to extend the life cycle of `request`. If it can be rvalue, we can move it. This could be check later. The change here is ok to me for now.